services/minespace-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -47,28 +56,71 @@ const documentColumns = [
   uploadDateColumn("upload_date"),
 ];
 
+const confirmationSubmission = (props) =>
+  props.isReviewSubmitStage &&
+  !props.applicationSubmitted &&
+  !props.formValues?.status_code && (
+    <Col span={24}>
+      <Card>
+        <>
+          <p>
+            <b>Confirmation of Submission</b>
+          </p>
+          <p>
+            <span>
+              <Checkbox
+                checked={props.confirmedSubmission}
+                onChange={props.setConfirmedSubmission}
+              />
+              &nbsp;&nbsp;
+            </span>
+            I understand that this application and supporting files are submitted on behalf of the
+            owner, agent or mine manager of this project.
+            <span style={{ color: "red" }}>*</span>
+          </p>
+        </>
+      </Card>
+      <br />
+    </Col>
+  );
+
 const renderInitialReport = (props) => (
   <Row>
     <Col span={24}>
       <Typography.Title level={4}>Initial Report</Typography.Title>
       <Typography.Paragraph>
         Select one or more incident types for this submission.
       </Typography.Paragraph>
-      <Form.Item label="Incident type(s)">
-        <Field
-          id="categories"
-          name="categories"
-          placeholder="Select incident type(s)"
-          component={renderConfig.MULTI_SELECT}
-          validate={[requiredList]}
-          data={props?.incidentCategoryCodeOptions}
+      {!props.isReviewSubmitStage && (
+        <Form.Item label="Incident type(s)">
```

Comment:
Is there a reason we can't just use the <Field> component and set disabled on that. Why do we need an <Input> when it is in review?